resources/views/components/pages/insight/index/newsletter.blade.php

```diff
@@ -3,7 +3,7 @@
         <h2 class="section-title mb-16">
             Subscribe to Our Newsletter
         </h2>
-        <div class="grid grid-cols-1 md:grid-cols-3 gap-0 rounded-xl border border-transparent bg-white uppercase">
+        <div class="grid grid-rows-3 sm:grid-cols-3  sm:grid-rows-1 gap-0 rounded-xl border border-transparent bg-white uppercase">
```

Comment:
use uk-grid don't use tailwindcss grid.

Reply:
<div class="uk-child-width-1-1 uk-child-wicth-1-3@s uk-grid-collapse rounded-xl border border-transparent bg-white uppercase" uk-grid>

Reply:
`<div class="uk-child-width-1-1 uk-child-width-1-3@s uk-grid-collapse rounded-xl border border-transparent bg-white uppercase" uk-grid>` as simple as that

Reply:
No need play around with rows and cols